pkg/sync/v9/data/Deployment.apps/openshift-infra/strimzi-cluster-operator.yaml

```diff
@@ -0,0 +1,95 @@
+      name: strimzi-cluster-operator
+      strimzi.io/kind: cluster-operator
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: strimzi
+        name: strimzi-cluster-operator
+        strimzi.io/kind: cluster-operator
+    spec:
+      containers:
+      - args:
+        - /opt/strimzi/bin/cluster_operator_run.sh
+        env:
+        - name: STRIMZI_NAMESPACE
+          value: '*'
+        - name: STRIMZI_FULL_RECONCILIATION_INTERVAL_MS
+          value: "120000"
+        - name: STRIMZI_OPERATION_TIMEOUT_MS
+          value: "300000"
+        - name: STRIMZI_DEFAULT_ZOOKEEPER_IMAGE
+          value: registry.redhat.io/amq7/amq-streams-kafka-22:1.2.0
+        - name: STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE
+          value: registry.redhat.io/amq7/amq-streams-kafka-22:1.2.0
+        - name: STRIMZI_DEFAULT_TLS_SIDECAR_KAFKA_IMAGE
+          value: registry.redhat.io/amq7/amq-streams-kafka-22:1.2.0
+        - name: STRIMZI_DEFAULT_TLS_SIDECAR_ZOOKEEPER_IMAGE
+          value: registry.redhat.io/amq7/amq-streams-kafka-22:1.2.0
+        - name: STRIMZI_KAFKA_IMAGES
+          value: |
+            2.1.1=registry.redhat.io/amq7/amq-streams-kafka-21:1.2.0
+            2.2.1=registry.redhat.io/amq7/amq-streams-kafka-22:1.2.0
+        - name: STRIMZI_KAFKA_CONNECT_IMAGES
+          value: |
+            2.1.1=registry.redhat.io/amq7/amq-streams-kafka-21:1.2.0
+            2.2.1=registry.redhat.io/amq7/amq-streams-kafka-22:1.2.0
+        - name: STRIMZI_KAFKA_CONNECT_S2I_IMAGES
+          value: |
+            2.1.1=registry.redhat.io/amq7/amq-streams-kafka-21:1.2.0
+            2.2.1=registry.redhat.io/amq7/amq-streams-kafka-22:1.2.0
+        - name: STRIMZI_KAFKA_MIRROR_MAKER_IMAGES
+          value: |
+            2.1.0=registry.redhat.io/amq7/amq-streams-kafka-21:1.2.0
+            2.2.1=registry.redhat.io/amq7/amq-streams-kafka-22:1.2.0
+        - name: STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE
+          value: registry.redhat.io/amq7/amq-streams-operator:1.2.0
+        - name: STRIMZI_DEFAULT_USER_OPERATOR_IMAGE
+          value: registry.redhat.io/amq7/amq-streams-operator:1.2.0
+        - name: STRIMZI_DEFAULT_KAFKA_INIT_IMAGE
+          value: registry.redhat.io/amq7/amq-streams-operator:1.2.0
+        - name: STRIMZI_DEFAULT_KAFKA_BRIDGE_IMAGE
+          value: registry.redhat.io/amq7/amq-streams-bridge:1.2.0
+        - name: STRIMZI_LOG_LEVEL
+          value: INFO
+        image: registry.redhat.io/amq7/amq-streams-operator:1.2.0
+        livenessProbe:
+          httpGet:
+            path: /healthy
+            port: 8080
+            scheme: HTTP
+          initialDelaySeconds: 10
+          periodSeconds: 30
+        name: strimzi-cluster-operator
+        readinessProbe:
+          httpGet:
+            path: /ready
+            port: 8080
+            scheme: HTTP
+          initialDelaySeconds: 10
+          periodSeconds: 30
+        resources:
+          limits:
+            cpu: "1"
```

Comment:
this is different to the original

Reply:
how so?

Reply:
in the yaml it was 100m I think, but it keeps getting changed to "1", so I have gone with that.

Reply:
I dont even see how `100m` could be right, considering request value is `200m` :man_shrugging:

Reply:
Do we want to make it 200m to make this guaranteed instead of burstable?